src/Extensions/Logging/Internal/OmexLoggerProvider.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Omex.Extensions.Logging
+{
+	internal class OmexLoggerProvider : ILoggerProvider, ISupportExternalScope
```

Comment:
>internal [](start = 1, length = 8)

comments, applies throughout

Reply:
It's an internal classes, I don't think it makes sense to have comments for them